### ISSUE TITLE
Burst of "Unable to send to wakeup socket!"

### ISSUE THREAD

**Issue by fcracker79** [dpkp/kafka-python#1837]
After upgrading from 1.4.4 to 1.4.6, we receive a burst of logs of "Unable to send to wakeup socket!".
Based on our metrics, clients are not consuming messages any more.
We solved the issue by restarting our clients.

**Comment by fcracker79** [comment]
At first I would suggest replacing

```
            except socket.error:
                log.warning('Unable to send to wakeup socket!')
```
with 
```
            except socket.error:
                log.warning('Unable to send to wakeup socket!', exc_info=True)
```
 just to dig into the issue.

**Comment by liho00** [comment]
Hi, i have the same problem...#1842
![Screenshot from 2019-06-10 17-01-51](https://user-images.githubusercontent.com/34411102/59184828-88a4ad80-8ba1-11e9-8752-1936b252ad49.png)

**Comment by dpkp** [comment]
I believe this is fixed in latest release, 1.4.7. Please reopen if issue persists.
